git.html.markdown

```diff
@@ -238,7 +241,7 @@ $ git clone -b master-cn https://github.com/adambard/learnxinyminutes-docs.git -
 ### commit
 
 Stores the current contents of the index in a new "commit." This commit contains
-the changes made and a message created by the user.
+the changes made and a message created by the user. *If you do not add the "-m" flag, git will throw you in to Vim or Nano, and your world will explode.*
```

Comment:
Ok, I will change that. The only reason I added the "world will explode"
comment is because I taught a workshop on using git and most users that
don't use it heavily forget to put the -m in and they are thrown in to vim
or nano, not even having a text editor specified as the default and they
freak out and don't know what to do and start smashing buttons.

On Wed, Oct 14, 2015 at 10:30 AM, Levi Bostian <notifications@github.com>
wrote:

> In git.html.markdown
> <https://github.com/adambard/learnxinyminutes-docs/pull/1487#discussion_r42000429>
> :
>
> > @@ -238,7 +241,7 @@ $ git clone -b master-cn https://github.com/adambard/learnxinyminutes-docs.git -
> >  ### commit
> >
> >  Stores the current contents of the index in a new "commit." This commit contains
> > -the changes made and a message created by the user.
> > +the changes made and a message created by the user. *If you do not add the "-m" flag, git will throw you in to Vim or Nano, and your world will explode.*
>
> Would be better if we said, *If you do not add the '-m' flag, git will
> open up a text editor for you to enter in the commit message. I have mine
> set for emacs for it is neither vim or nano...and my world does not
> explode, emacs makes me happy :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adambard/learnxinyminutes-docs/pull/1487/files#r42000429>
> .
>